### DISCUSSION TITLE
스터디 운영안내

### DISCUSSION THREAD

**Discussion by chaneeii** [WWHigh/WWDC-STUDY#19] [Announcements]
### 운영방식
> 💻 WWDC 세션 시청 후 정리한 내용을 Github Issue 및 오프라인으로 공유

- 스터디 인원을 2개 그룹으로 나누어 각 차시별 격주로 각자 주제에 대해 오프라인 공유 
  - WWDC 세션 영상을 시청 후, `Github Issue` 에 labelling 하여 공유
  - `주 1회` 오프라인으로 각자 학습한 주제에 대해 `오프라인` 공유 

- 차시별 주제 선정방식은 아래와 같음
  - 모든 차시 공통주제 :  Swift / SwiftUI 및 프레임워크
  - 차시별 선정주제 : 팀원 협의에 의해 결정

### 스터디인원

| 👩‍💻 Name 🧑‍💻    | 🔗 Github 🔗                           
|----------------	|------------------------------------
| 김승창 (Charlie)| [LentoAssai](https://github.com/LentoAssai)         
| 김영우 (Poding)| [Yeongwoo-Poding](https://github.com/Yeongwoo-Poding) 
| 김의성 (Youth)| [kimscastle](https://github.com/kimscastle)                              
| 민채호 (Avo)  | [Chaeho-Min](https://github.com/Chaeho-Min)                                  
| 박종호 (HoJongE) | [HoJongPARK](https://github.com/HoJongPARK)                                
| 박준홍 (Noasdaq) | [feldblume5263](https://github.com/feldblume5263)                             
| 이재용 (Woody) | [wody-d](https://github.com/wody-d)   
| 임영후 (Asher) | [Asher3576](https://github.com/Asher3576)
| 정찬희 (Avery) |[chaneeii](https://github.com/chaneeii)                                           
| 한택환 (Taek) |[TaekH](https://github.com/TaekH)

**Comment by samsung-ga** [comment]
스터디 운영 방식 좋은 것 같습니다 :) 동의합니다
